runelite-client/src/main/java/net/runelite/client/plugins/diaryprogress/DiaryProgressPlugin.java

```diff
@@ -75,7 +75,8 @@ public void updateDiaryStatus(GameTick event)
 			{
 				for (Varbits varbits : entry.getVarbits())
 				{
-					if ((entry.getIndex() != 0 && client.getSetting(varbits) == 1) || client.getSetting(varbits) == 2)
+					int value = client.getSetting(varbits);
```

Comment:
These 2 lines should be part of the previous commit

Reply:
The two lines have been moved back to the previous commit